.travis.yml

```
@@ -19,6 +19,10 @@ before_install:
 install:
   - travis_retry composer install --no-interaction --prefer-dist --no-suggest
 
+before_script:
+  - cp .env.example .env
+  - php artisan key:generate
+
```

```suggestion
- php artisan config:clear
- php artisan serve &

```

This may be due to the server not running.